numpy/core/multiarray.py

```diff
@@ -1188,6 +1194,12 @@ def unpackbits(a, axis=None, count=None):
 
         .. versionadded:: 1.17.0
 
+    order : 'big' or 'little', only the first letter is checked
+        The order of the returned bits. The default is the common standard
+        where 3 => [0, 0, 0, 0, 0, 1, 1]
```

Comment:
Shouldn't there be 8 items in this list, not 7?

I'm not sure I'd call this the common standard - I frankly find it bizarre that `unpackbits(1)[0]` does not refer to "bit 0" of x, `.

Could instead compare to binary literal syntax, stating that  `0b00000011 => [0,0,0,0,0,0,1,1]`

Reply:
Seeing the comment, I remember being bitten by exactly this (which is of course why "little" will be useful!).

Maybe the docstring can be agnostic about which is "standard" - both have advantages. I like the idea of adding `3 = 0b00000011 => [...]`

Reply:
adopted